xmake/rules/qt/xmake.lua

```diff
@@ -249,10 +255,10 @@ rule("qt.quickapp_static")
         elseif target:is_plat("wasm") then
             plugins.QWasmIntegrationPlugin = {linkdirs = "plugins/platforms", links = {"qwasm"}}
             if qt_sdkver then
-                if qt_sdkver:lt("6.0") then
-                    table.join2(frameworks, "QtEventDispatcherSupport", "QtFontDatabaseSupport", "QtEglSupport")
-                else
+                if qt_sdkver:ge("6.0") then
                     table.join2(frameworks, "QtOpenGL")
+                else
+                    table.join2(frameworks, "QtEventDispatcherSupport", "QtFontDatabaseSupport", "QtEglSupport")
                 end
             end
```

Comment:
you should use `qt_sdkver and qt_sdkver:ge("6.0")`, otherwise frameworks will be missing if qt_sdkver == nil

```
            if qt_sdkver and qt_sdkver:ge("6.0") then
                    table.join2(frameworks, "QtOpenGL")
                else
                    table.join2(frameworks, "QtEventDispatcherSupport", "QtFontDatabaseSupport", "QtEglSupport")
                end
```

Reply:
Why qt version may be unknown? It's better to throw an error when version of qt is undetermined.

Reply:
you can add `assert(qt_sdkver, "Qt SDK version not found!")` in 

https://github.com/xmake-io/xmake/blob/abd463ca2191769b6ee6012f4b823ce4c3c8623f/xmake/rules/qt/xmake.lua#L131

then remove `if qt_sdkver then`